src/test/java/org/cactoos/list/ImmutableTest.java

```diff
@@ -38,9 +42,24 @@
  * @checkstyle MagicNumberCheck (500 lines)
  * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
  */
-@SuppressWarnings("PMD.TooManyMethods")
+@SuppressWarnings({"PMD.TooManyMethods", "PMD.AvoidDuplicateLiterals"})
 public class ImmutableTest {
 
+    @Test
+    public void testImmutability() {
```

Comment:
@iakunin please, change the test name for other more meaningful.

Reply:
@fabriciofx changed to `innerListMustNotBeChanged()`